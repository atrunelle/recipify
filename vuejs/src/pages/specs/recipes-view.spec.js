import { mount } from '@vue/test-utils';
import RecipesView from '@/pages/recipes-view';

describe('Component: recipes view', () => {
  let component;

  beforeEach(() => {
    component = mount(RecipesView, {
      stubs: {
        'recipes-list': '<div></div>',
      },
    });
  });

  it('should match snapshot', () => {
    expect(component.html()).toMatchSnapshot();
  });
});
