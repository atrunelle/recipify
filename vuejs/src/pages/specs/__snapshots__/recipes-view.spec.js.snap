// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: recipes view should match snapshot 1`] = `
<div class="layout column">
    <h2>List of recipes you created</h2>
    <div></div>
</div>
`;
